static_report/README.md

```diff
@@ -13,13 +13,15 @@ This is <span style="color: red">REQUIRED</span> for serving your apps, as it de
 1. Install Python packages: `pip install -r requirements.txt` (now you are running the CLI from the python source code)
 1. Add a local data source (see [this quickstart step](https://docs.piperider.io/quick-start#prepare-sqlite-database))
 1. Run `piperider init` (creates `.piperider/` dir)
-1. Generate your Single and Comparison reports (`piperider run` for single; `piperider compare-reports` for comparison)
+1. Generate your Single and Comparison reports
+   - `piperider run` for single reports (generates `run.json`);
+   - `piperider compare-reports --debug` (generates `comparison_data.json`)
 1. Now, you are ready. Proceed to the FE side. `cd static_report`
 
 ### Installation
 
 ```sh
-$ npm install
+$ yarn  # NOTE to `npm` users: do not commit package-lock.json
```

Comment:
We can add `package-lock.json` to `.gitignore`?

Reply:
true